docs/samples/tensorflow/README.md

```diff
@@ -57,4 +57,57 @@ Expected Output
     ]
 * Connection #0 to host 34.83.190.188 left intact
 }%
+```
+
+If you stop making requests to the application, you should eventually see that your application scales itself back down to zero. Watch the pod until you see that it is `Terminating`. This should take approximately 90 seconds.
+
+    ```
+    kubectl get pods --watch
+    ```
+
+    Note: To exit the watch, use `ctrl + c`.
+
+## Canary Rollout
+
+To test a canary rollout, you can use the tensorflow-canary.yaml 
+
+Apply the CRD
+```
+kubectl apply -f tensorflow-canary.yaml 
+```
+
+To verify if your traffic split percenage is applied correctly, you can use the following command:
+
+```
+kubectl get kfservices
+NAME             URL                                  DEFAULT TRAFFIC   CANARY TRAFFIC   AGE
+flowers-sample   flowers-sample.default.example.com   90                10               48s
+```
+
+If you are using the [Knative CLI (knctl)](#knative-cli), run the following command
+
+```
+knctl revision list 
+Revisions
+
+Service                 Name                          Tags  Annotations                                                 Conditions  Age  Traffic  
+flowers-sample-canary   flowers-sample-canary-6kpt6   -     autoscaling.knative.dev/class: kpa.autoscaling.knative.dev  4 OK / 5    40m  10% -> flowers-sample.default.example.com  
+                                                            autoscaling.knative.dev/target: "1"                                            
+flowers-sample-default  flowers-sample-default-l9c24  -     autoscaling.knative.dev/class: kpa.autoscaling.knative.dev  4 OK / 5    40m  90% -> flowers-sample.default.example.com  
+                                                            autoscaling.knative.dev/target: "1"  
+```
+
+### Knative CLI:
```

Comment:
This instructions should be in the developer guide instead of TF specific.

Reply:
done